libswirl/utils/glinit/glx/glx.cpp

```diff
@@ -3,6 +3,7 @@
 #include "utils/glwrap/gl3w.h"
 
 #include <GL/glx.h>
+#include <GL/glxext.h>
```

Comment:
`glext` was not used on purpose, is there some reason to have it as a dependency?

Reply:
To remove this:
https://github.com/reicast/reicast-emulator/blob/6c440b51ffff03866d8e0eecd41316ad915df488/libswirl/utils/glinit/glx/glx.cpp#L87-L89

Reply:
The definitions are guaranteed to not change - this was on purpose to not depend on `glext.h`. Please do a git-rebase and undo this change

Reply:
Change reverted, code rebased